code/modules/clothing/glasses/_glasses.dm

```diff
@@ -281,6 +281,33 @@
 	..()
 	user.cure_blind("blindfold_[REF(src)]")
 
+/obj/item/clothing/glasses/sunglasses/blindfold/white

```

Comment:
Fun fact cult stuns will still work against blinding people do to being a magic light that appears in their head!

Reply:
What spell? Do all cult spells rely on sight?

Reply:
The clocks use a blinding light from clock slab.
And Blood cult use a sorta stun spell as well that is a white flash of light.
No matter what, even well blind you get stunned by these attacks, no cult spell I know of relys on sight nor relys on a targets sight